common/src/app/common/files/repair.cljc

```diff
@@ -68,7 +68,19 @@
 
 (defmethod repair-error :child-not-found
   [_ {:keys [shape page-id args] :as error} file-data _]
-  (log/info :hint "Repairing shape :child-not-found" :id (:id shape) :name (:name shape) :page-id page-id)
+  (let [repair-shape
+        (fn [parent-shape]
+          (log/debug :hint "  -> Remove children from" :shapes (:child-id args))
+          (update parent-shape :shapes (fn [shapes]
+                                         (filterv #(not= (:child-id args) %) shapes))))]
+    (log/info :hint "Repairing base shape :child-not-found" :id (:child-id args) :name "page" :page-id page-id)
```

Comment:
I prefer to stick to standard message here, and give more info in log/debug above
`(log/info :hint "Repairing shape :child-not-found" :id (:id shape) :name (:name shape) :page-id page-id)`